README.md

```diff
@@ -1,2 +1,44 @@
 # comp110-worksheet-B
 Base repository for COMP110 worksheet B
+
+```
+enter terminal
+
+wordArray <- List of words
+secretWord <- randomly select word form wordArray
+attampts <- 4
+compleat <- false
+
+Display all words form wordArray and add random charitors inbetween words all as sellectable buttons
+
+while attampts >= 0:
+
+	Wait for user to sellect and submit a word
+	guesedWord <- submited word
+	
+	likeness <- 0
+	
+	for n <- 0; n < secretWord Length; n++:
```

Comment:
I'd argue that C-style `for` loops are not the most readable choice for pseudocode -- I'd generally write this as something like
```
for n = 0, 1, ..., secretWordLength
```